js/yasgui.local.js

```diff
@@ -25,4 +1415,262 @@ LIMIT 100`;
+          var tagCheckbox = document.createElement('input');
+          tagCheckbox.classList.add('tagCheckbox');
+          tagCheckbox.id = 'tag_' + tagId;
+          tagCheckbox.setAttribute('name', 'tagCheckbox');
+          tagCheckbox.setAttribute('type', 'checkbox');
+          tagCheckbox.setAttribute('value', tag);
+          tagHtml.appendChild(tagCheckbox);
+          var tagLabel = document.createElement('label');
+          tagLabel.appendChild(document.createTextNode(tag));
+          tagLabel.setAttribute('for', 'tag_' + tagId);
+          tagHtml.appendChild(tagLabel);
+          tagsListHtml.appendChild(tagHtml);
+          tagCheckbox.addEventListener('change', refreshList);
+        }
+      });
+    }
+  });
+  
+  //chargement des exemples depuis le fichier json en appliquant les filtres
+  refreshList();
+  
+  var executeBt = document.getElementById('executeExample');
+  executeBt.addEventListener('click', function(){
+    var selected = examplesData[getSelectedExample()];
+    if(selected === undefined) return;    
+    var newTab = yasgui.addTab();
+    newTab.rename(selected.title);
+    var query = "# Description : \n# " + selected.description + "\n\n" + selected.content;
+    newTab.setQuery(query);
+    var endpoint = selected.endpoint;
+    if(endpoint !== undefined) {
+      newTab.setEndpoint(endpoint);
+    }else{
+      newTab.setEndpoint(YASGUI.defaults.yasqe.sparql.endpoint);
+    }
+    hidePopup(closeBt.closest('.popupContainer'));
+    newTab.yasqe.query();
+  });
+  
+  var closeBt =  document.getElementById('closePopup');
+  closeBt.addEventListener('click', function(){
+    hidePopup(closeBt.closest('.popupContainer'));
+  });
+  
+  
+  //affichage des exemples lors du click sur le bouton "voir des exemples"
+  document.getElementById('showExamples').addEventListener('click', function(){ 
+    if(examplesPopup.classList.contains('showPopup')){
+      hidePopup(examplesPopup);
+    }else{
+      showPopup(examplesPopup);
+    }
+  });
+  
+  //affichage de la selection de tags
+  document.querySelectorAll('.dropdown-toggle').forEach((e) =>{
+    e.addEventListener('click', function(){
+      var stateOpen = e.parentNode.classList.contains('dropdown-open');
+      closeDropdowns();
+      if(!stateOpen){
+        e.parentNode.classList.add('dropdown-open');
+      }
+    });
+  });
+  
+  //recherche
+  document.getElementById('searchExamples').addEventListener('keyup', function(){
+    refreshList();
+  });
+  
+  // detection du clavier
+  document.addEventListener('keyup', function(event){
+    //appui sur la touche echap
+    if(event.code === "Escape"){
+      //fermeture de la selection des tags
+      if(closeDropdowns()) { return; }
+      //fermeture de la pop-up
+      if(examplesPopup.classList.contains("showPopup")){
+        hidePopup(examplesPopup);
+        return;
+      }
+    }
+  });
+  
+  //interractions de fermeture au clic n imorte ou sur la page
```

Comment:
```suggestion
  //interactions de fermeture au clic n'importe où sur la page
```

Ah, tu ne mets pas d'accent dans le code?
Normalement, avec de l'UTF-8 tu ne risques rien.